src/Coinbase/Spot/API/FeeEstimate.jl

```diff
@@ -0,0 +1,73 @@
+module FeeEstimate
+
+export FeeEstimateQuery,
+    FeeEstimateData,
+    fee_estimate
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Coinbase
+using CryptoAPIs: Maybe, APIsRequest
+
+Base.@kwdef struct FeeEstimateQuery <: CoinbasePrivateQuery
+    currency::Maybe{String} = nothing
+    crypto_address::Maybe{String} = nothing
+    network::Maybe{String} = nothing
+
+    timestamp::Maybe{DateTime} = nothing
+    signature::Maybe{String} = nothing
+end
```

Comment:
`FeeEstimateQuery` structure must be mutable, as in `CryptoAPIs.request_sign!` 2 fields of the structure will be changed